test/views/html/pages/SupportForCoronavirusViewSpec.scala

```diff
@@ -39,7 +39,7 @@ class SupportForCoronavirusViewSpec extends ChatViewBehaviours {
             val doc = asDocument(createView())
             val nav = doc.getElementById("proposition-menu")
             val span = nav.children.first
-            span.text mustBe messages("global.nav.title")
+            span.text mustBe messages("Ask HMRC’s digital assistant")
```

Comment:
should this not be global.nav.va.title

Reply:
No, as the VA guys want a different title. Look at tick DEP-299 Tim explains he could not get that working on the prototype